templates/registration/login.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{%  block content %}
+<h1>Login</h1>
+  {% if form.errors %}
+  <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">
```

Comment:
The form's action URL is set to 'login'. Ensure that the URL pattern for login is correctly named 'login' in your URL configuration.